github/github-accessors_test.go

```diff
@@ -0,0 +1,18304 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
```

Comment:
Should the copyright year get updated based on when these files are (re)generated?

Reply:
Good question, but the answer is no. Once a file is created, the copyright year remains unmodified.
(At least for this repo, that is the instruction given from the Google Open Source team.)

Reply:
Oh!  I see your point now.  This is a new test file.

OK, I'm going to make an executive decision and keep the two sync'd to the same 2017 year.

Thank you, @nightlark .